src/App.jsx

```diff
@@ -1,13 +1,197 @@
+import React, {useState} from 'react';
+import './App.css';
+import Counter from "./Counter/Counter.jsx";
+import Button from "./Button/Button.jsx";
 
-import './App.css'
 
 function App() {
+    const [count1, setCount1] = useState(0);
+    const [count2, setCount2] = useState(0);
+    const [count3, setCount3] = useState(0);
+    const [count4, setCount4] = useState(0);
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [age, setAge] = useState(0);
+    const [zipCode, setZipCode] = useState('');
+    const [delivery, setDelivery] = useState('');
+    const [time, setTime] = useState('');
+    const [comment, setComment] = useState('');
+    const [check, setCheck] = useState(false);
+
+    const increase = (setter, count) => () => setter(count + 1);
+    const decrease = (setter, count) => () => { if (count > 0) setter(count - 1); };
```

Comment:
```suggestion
    const decrease = (setter, count) => () => setter(count - 1);
```
Je kan het met het disabled attribuut van het button element ervoor zorgen dat deze functie niet eens uitgevoerd wordt als de counter 0 is